<ui version="4.0" >
 <author></author>
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>transformTrans</class>
 <widget class="QWidget" name="transformTrans" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Transform Transaction</string>
  </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="XLabel" name="_transDateLit" >
            <property name="text" >
             <string>Transaction Date:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>_transDate</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DLineEdit" name="_transDate" />
          </item>
          <item>
           <widget class="XLabel" name="_usernameLit" >
            <property name="text" >
             <string>Username:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>_username</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="XLabel" name="_username" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="ItemCluster" name="_item" />
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="_close" >
            <property name="text" >
             <string>&amp;Close</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_post" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>&amp;Post</string>
            </property>
            <property name="autoDefault" >
             <bool>true</bool>
            </property>
            <property name="default" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="XLabel" name="_warehouseLit" >
        <property name="text" >
         <string>Site:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="WComboBox" name="_warehouse" />
      </item>
      <item row="1" column="2" >
       <widget class="XLabel" name="_fromBeforeQty" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="XLabel" name="_fromAfterQty" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="XLabel" name="_beforeLit" >
        <property name="text" >
         <string>Before</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="XLabel" name="_afterLit" >
        <property name="text" >
         <string>After</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="XLabel" name="_targetLit" >
        <property name="text" >
         <string>Target Item Number:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="XComboBox" name="_target" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="XLabel" name="_qtyLit" >
        <property name="text" >
         <string>Qty. to Transform:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="XLineEdit" name="_qty" />
      </item>
      <item row="5" column="0" >
       <widget class="XLabel" name="_documentNumLit" >
        <property name="text" >
         <string>Document #:</string>
        </property>
        <property name="buddy" >
         <cstring>_documentNum</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3" >
       <widget class="QLineEdit" name="_documentNum" />
      </item>
      <item row="2" column="2" >
       <widget class="XLabel" name="_toBeforeQty" >
        <property name="text" >
         <string/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="XLabel" name="_toAfterQty" >
        <property name="text" >
         <string/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="XLabel" name="_descrip1" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2" >
       <widget class="XLabel" name="_descrip2" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="XLabel" name="_sourceLit" >
        <property name="text" >
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="XTreeWidget" name="_source" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="XLabel" name="_notesLit" >
        <property name="text" >
         <string>Notes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="XTextEdit" name="_notes" />
      </item>
     </layout>
    </item>
   </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemCluster</class>
   <extends>QWidget</extends>
   <header>itemcluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WComboBox</class>
   <extends>XComboBox</extends>
   <header>wcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XLabel</class>
   <extends>QLabel</extends>
   <header>xlabel.h</header>
  </customwidget>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>XTextEdit</class>
   <extends>QTextEdit</extends>
   <header>xtextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_transDate</tabstop>
  <tabstop>_item</tabstop>
  <tabstop>_warehouse</tabstop>
  <tabstop>_target</tabstop>
  <tabstop>_qty</tabstop>
  <tabstop>_documentNum</tabstop>
  <tabstop>_source</tabstop>
  <tabstop>_notes</tabstop>
  <tabstop>_post</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <includes>
  <include location="global" >parameter.h</include>
  <include location="global" >QStatusBar</include>
  <include location="global" >QMessageBox</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>transformTrans</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_item</sender>
   <signal>warehouseIdChanged(int)</signal>
   <receiver>_warehouse</receiver>
   <slot>setId(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_item</sender>
   <signal>privateIdChanged(int)</signal>
   <receiver>_warehouse</receiver>
   <slot>findItemsites(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
